src/bbmustache.erl

```diff
@@ -124,7 +124,8 @@
 
 -type compile_option() :: {key_type, atom | binary | string}
                        | raise_on_context_miss
-                       | {escape_fun, fun((binary()) -> binary())}.
+                       | {escape_fun, fun((binary()) -> binary())}
+                       | {pre_processor, fun((any()) -> any())}.
```

Comment:
It may be hard to understand means..
It is especially difficult to understand the execution timing.

It is easier to understand if option name is `value_serializer`.

```erl
Serializer = proplists:get_value(value_serializer, State#?MODULE.options, fun default_value_serializer/1)
iolist_to_binary(Serializer(RawData))
```

- convert to `default_value_serializer` from `to_iodata`


usage:
```erl
my_serializer(Term) when is_map(Term) ->
  % convert this.
my_serializer(Term) ->
  bbmustache:default_value_serializer(Term).
```